modules/social_features/social_group/modules/social_group_secret/config/install/group.type.secret_group.yml

```diff
@@ -0,0 +1,10 @@
+langcode: en
+status: true
+dependencies: {  }
+id: secret_group
+label: 'Secret group'
+description: 'This is a secret group.'
```

Comment:
This is displayed on the group type selection screen so we need some more explanation here of what this means.

Examples for the other group types are:

`This is a public group. Users may join without approval and all content added in this group will be visible to all community members and anonymous users. `
` This is an open group. Users may join without approval and all content added in this group will be visible to all community members. `
` This is a closed group. Users can only join by invitation and all content added in this group will be hidden for non members. `

Reply:
@NatashaSchon what text needs to use here?